src/devices/schou_72543_rain.c

```diff
@@ -111,18 +111,18 @@ static int schou_72543_rain_decode(r_device *decoder, bitbuffer_t *bitbuffer)
     deviceID        =   (b[0] << 8 ) | b[1];
     isBatteryLow    =   (b[2] & 0x80) == 1;                  // if one, battery is low
     isMessageRepeat =   (b[2] & 0x40) == 1;                  // if one, message is a repeat
-	messageCounter  =   (b[2] & 0x0e) >> 1;                  // 3 bit counter (rather than 4 bit incrementing by 2 each time
+    messageCounter  =   (b[2] & 0x0e) >> 1;                  // 3 bit counter (rather than 4 bit incrementing by 2 each time
     rain_mm         =  ((b[4] << 8 ) & b[3]) / 10.0f;
-	temp_F          = (((b[6] << 8 ) & b[5]) / 10.0f) - 90;
+    temp_F          = (((b[6] << 8 ) & b[5]) / 10.0f) - 90;
 
     /* clang-format off */
     data = data_make(
             "model",          "",             DATA_STRING, "Schou 72543 rain sensor",
```

Comment:
model needs to be just "Schou-72543"

Reply:
Thank you @zuckschwerdt for the comments. I seriously appreciate the support!
And I am sorry for the amount of commits. I still did not get the "squish" i think it is called of multiple- to one commit.

Reply:
Happy to help. No need to squash. We'll do it on merge anyway.